wagon.py

```diff
@@ -451,8 +451,7 @@ def _get_pip_path(venv=None):
     if venv:
         return os.path.join(_get_env_bin_path(venv), pip)
     else:
-        return os.path.join(
-            os.path.dirname(sys.executable), 'scripts' if IS_WIN else '', pip)
+        return os.path.join(os.path.dirname(sys.executable), pip)
```

Comment:
Um... this can pose a problem if pip is not in the same dir as python, which can happen.

Reply:
That is, if pip really is always in the same dir on Windows, I'd add an if there since on Linux/OSx it's not the case.

Reply:
@nir0s , if you compare my update to the previous revision, you'll see that there's no change for Linux platforms. The old line:

`os.path.dirname(sys.executable), 'scripts' if IS_WIN else '', pip)`

So my change only affects Windows.
The old code already assumes that `pip` is at the same dir as `python`.

Reply:
My bad.